controllers/grafanafolder_controller.go

```diff
@@ -218,10 +219,8 @@ func (r *GrafanaFolderReconciler) SetupWithManager(mgr ctrl.Manager, stop chan b
 		go func() {
 			for {
 				select {
-				case <-stop:
```

Comment:
You should most likely listen for `<-ctx.Done()` now

Reply:
Didn't think I needed it due to ctx now including stop.
I re added it any way. 
Thanks for the check @simongottschlag .